### DISCUSSION TITLE
Unlisted from CDC Forecasts

### DISCUSSION THREAD

**Discussion by caominhduy** [reichlab/covid19-forecast-hub#2793] [Q&A] [answered]
Hi,

We are [Pandemic Central](itsonit.com) and we have submitted weekly projections since August 2020.  Currently, our initiative's alias has been unlisted from [COVID-19 Forecasts: Cases](https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/forecasts-cases.html), although as far as we can tell, our latest data was still eligible and used for ensemble modelling.

Was this a mistake or did we miss any special requirement?

Thank you.

**Comment by ydhuang28** [comment, marked as answer]
Hi! Your submissions have only included county-level forecasts. We do not aggregate forecasts from county level to state level, but you can see from [this PDF published by the CDC](https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/Consolidated-Cases-Forecasts-2021-01-25.pdf) (specifically, see Autauga County, Alabama) that your model has not been excluded.

**Reply by caominhduy** [reply]
Thanks. So when it says "forecasts included in the ensembles are displayed below," it means *only* forecasts included in the *state-level* ensembles are displayed below?

**Reply by ydhuang28** [reply]
The CDC has its own criteria for whether a model gets listed under that designation on that page, so we can't say for sure. You can reach out to them directly for a definite answer.

**Reply by caominhduy** [reply]
Thanks!
